lambdas/api-handler/src/handlers/__tests__/patch-letter.test.ts

```diff
@@ -51,257 +53,290 @@ describe('patchLetter API Handler', () => {
-      }
+        "nhsd-supplier-id": "supplier1",
+        "nhsd-correlation-id": "correlationId",
+        "x-request-id": "requestId",
+      },
     });
     const context = mockDeep<Context>();
     const callback = jest.fn();
 
     const patchLetterHandler = createPatchLetterHandler(mockedDeps);
     const result = await patchLetterHandler(event, context, callback);
 
-    expect(mockedProcessError).toHaveBeenCalledWith(new ValidationError(errors.ApiErrorDetail.InvalidRequestBody), 'correlationId', mockedDeps.logger);
+    expect(mockedProcessError).toHaveBeenCalledWith(
+      new ValidationError(errors.ApiErrorDetail.InvalidRequestBody),
+      "correlationId",
+      mockedDeps.logger,
+    );
     expect(result).toEqual(expectedErrorResponse);
   });
 
-  it('returns error if path letterId and body letterId do not match', async () => {
+  it("returns error if path letterId and body letterId do not match", async () => {
     const event = makeApiGwEvent({
-      path: '/letters/id2',
+      path: "/letters/id2",
       body: requestBody,
-      pathParameters: {id: 'id2'},
+      pathParameters: { id: "id2" },
       headers: {
-        'nhsd-supplier-id': 'supplier1',
-        'nhsd-correlation-id': 'correlationId',
-        'x-request-id': 'requestId'
-      }
+        "nhsd-supplier-id": "supplier1",
+        "nhsd-correlation-id": "correlationId",
+        "x-request-id": "requestId",
+      },
     });
     const context = mockDeep<Context>();
     const callback = jest.fn();
 
     const patchLetterHandler = createPatchLetterHandler(mockedDeps);
     const result = await patchLetterHandler(event, context, callback);
 
-    expect(mockedProcessError).toHaveBeenCalledWith(new ValidationError(errors.ApiErrorDetail.InvalidRequestLetterIdsMismatch), 'correlationId', mockedDeps.logger);
+    expect(mockedProcessError).toHaveBeenCalledWith(
+      new ValidationError(
+        errors.ApiErrorDetail.InvalidRequestLetterIdsMismatch,
+      ),
+      "correlationId",
+      mockedDeps.logger,
+    );
     expect(result).toEqual(expectedErrorResponse);
   });
 
-  it('returns error if unexpected error is thrown', async () => {
+  it("returns error if unexpected error is thrown", async () => {
     const event = makeApiGwEvent({
-      path: '/letters/id1',
-      body: 'somebody',
-      pathParameters: {id: 'id1'},
+      path: "/letters/id1",
+      body: "somebody",
+      pathParameters: { id: "id1" },
       headers: {
-        'nhsd-supplier-id': 'supplier1',
-        'nhsd-correlation-id': 'correlationId',
-        'x-request-id': 'requestId'
-      }
+        "nhsd-supplier-id": "supplier1",
+        "nhsd-correlation-id": "correlationId",
+        "x-request-id": "requestId",
+      },
     });
     const context = mockDeep<Context>();
     const callback = jest.fn();
 
-    const error = 'Unexpected error';
-    const spy = jest.spyOn(JSON, 'parse').mockImplementation(() => {
+    const error = "Unexpected error";
```

Comment:
```suggestion
    const error = new Error();
```
Would this avoid the noise around suppressing the error?